backend/data/migrations/20200321012509_create_orders_table.js

```diff
@@ -0,0 +1,37 @@
+module.exports.up = function (knex) {
+  return knex.schema.createTable("orders", (orders) => {
+    orders.increments();
+    orders.boolean("complete").notNullable().defaultTo(false);
```

Comment:
Maybe state machine instead? Field `status` with enumeration to 
```
CREATED
COMPLETED
CANCELLED
...
```

Reply:
Check